#goal definition
int32 result_chooser
---
#result definition
float32 timefull
float32 timeempty

---
#feedback
float32 SOC_percent
float32 SOC_mAh
string status

